controllers/storagesystem_controller.go

```diff
@@ -209,6 +211,46 @@ func (r *StorageSystemReconciler) addReferenceToRelatedObjects(instance *odfv1al
 // SetupWithManager sets up the controller with the Manager.
 func (r *StorageSystemReconciler) SetupWithManager(mgr ctrl.Manager) error {
 
+	// ========== START HACK ==========
+	// TODO: This is a hack to work around the absense of StorageCluster CRs.
+	// A better fix should come in ASAP.
```

Comment:
Would be good to explain:
* why this fix is bad
* what a better fix would be
* what prevents us from doing the better fix and forces us do to the bad fix (for now)

Reply:
Yes, I am only requesting that this be explained somewhere. Commit message or PR description (or both).